### ISSUE TITLE
Column marking/editing

### ISSUE THREAD

**Issue by diagonal-III** [subethaedit/SubEthaEdit#68]
It would be nice to be able to do column marking/editing.

Two possible modes: 

**absolute mode** simply uses the character position for marking so you could mark, for instance, column 1..6 of lines 3..8 and copy/cut/delete it or paste elsewhere (again per column pasted so if insert point would be at character 10, it would insert the copied text in every line at position 10.

**marker mode** would not mark per column, but per marker.  This would allow, for instance, working on CSV files which would then allow record edits per CSV field by setting the relevant marker to whatever field separator in use (this can get tricky, sometimes it's a comma but it would then intelligently have to identify text fields enclosed in quotes).  This mode would help rearranging field sequences, for instance.  

The above "marker mode" hasn't been done anywhere else (AFAIK), but that may also be because it has a fairly limited audience - it may not be worth the time.  Column editing, however, is available in a number of editors.

**Comment by monkeydom** [comment]
Agreed. SubEthaEdit needs an improvement over its old blockediting to a better concept like multiple insertion points and column selection.
